stages/openmvs.py

```diff
@@ -78,7 +79,7 @@ def process(self, args, outputs):
 
             log.ODM_INFO("Computing sub-scenes")
             config = [
-                "--sub-scene-area 10000",
+                "--sub-scene-area 660000",
```

Comment:
16 gigs feels, while not unreasonable, not as accessible as 8GB memory ceiling. We get so many folks on the community forum showing up with 8GB on their hardware. Is there large enough a disadvantage to trigger a split at a smaller sub-scene size?

Reply:
Not really, we can probably lower it. 16GB is an approximation, I'm still trying to figure out exactly how that parameter affects the splits, it's a bit non-intuitive (so not sure by what unit I have to scale it to half memory requirements). Accepting better value suggestions.

Reply:
To further the 8GB recommendation, WSL uses half the physical memory by default as the max it will allocate to the WSL2 instance... With many machines shipping with 16GB as premium, that means that 8GB is a more comfortable lower limit. It would also make processing on a single Raspberry Pi 8GB tenable, which is cool, if impractical if not clustered.